tests/terraform/module_loading/test_tf_module_finder.py

```diff
@@ -24,6 +24,12 @@ def test_module_finder(self):
             else:
                 self.assertIsNone(m.version)
 
+    def test_module_finder_ignore_comments(self):
+        modules = find_modules(self.get_src_dir())
+        module_list = list(map(lambda mod: mod.module_link, modules))
+        for m in module_list:
+            self.assertNotIn(m, ["comment/local_module", "terraform-aws-modules/s3-bucket/ignore"])
```

Comment:
@kartikp10 `terraform-aws-modules/s3-bucket/ignore` is not used anywhere... In general, let's use a positive case test here and not negative test case to ensure stability 🙏

Reply:
Fixed. I hope that makes sense.